brave/brave-core
# Move iOS recovery code into Backup and Restore modal

Resolves https://github.com/brave/brave-browser/issues/16954

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Launch Brave
- Visit brave://rewards
- Verify that QR code section doesn't appear in Settings page
- Click gear icon
- Select the Restore tab in the modal
- Verify that QR code link looks as expected
- Click the QR code link
- Verify that appropriate QR code displays
- Verify that QR code functions as expected
